docker-compose.yml

```diff
@@ -27,8 +29,8 @@ services:
     networks:
       - dlme
     # For running in dev
-    volumes:
-      - ".:/opt/"
+#    volumes:
+#      - ".:/opt/"
```

Comment:
If we don't have this, then if we change a file on the fs, isn't not changed on the server, right?

Reply:
Yes. So if someone wants to run the app in a dev mode, the appropriate lines could be uncommented. But open to other suggestions.

Reply:
I would prefer that a change like this should be made in a separate pull request that is about changing the environment.  I think the change you want to make here is adding these lines to `config/environments/development.rb`


```ruby
config.active_job.queue_adapter     = :sidekiq
config.active_job.queue_name_prefix = "dlme_#{Rails.env}"
```

Reply:
Agree.